.github/workflows/fetch.yml

```diff
@@ -4,8 +4,9 @@ on:
     - cron: '30 5,12,17 * * *'
   push:
 jobs:
-  docker-build:
+  deploy:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

Comment:
probably want to make this a push condition ?

Reply:
I don't think so? This is a deploy condition. I don't want ever to have a deploy from a branch that is not main. Not sure what would happen if more than one branch starts a cronjob at the same time? maybe a race condition?

Reply:
schedule will always run on the main branch.